run-build.ps1

```diff
@@ -32,7 +32,7 @@ if ($Sign) {
 $Parameters = "$Parameters -WarnAsError `$$WarnAsError"
 
 try {
-    Set-PSDebug 2
+    Set-PSDebug -Trace 2
```

Comment:
🤦🏻‍♂️ sorry

Reply:
still not working, I think I need to set it separately in build.cmd or something